### PR TITLE
[WIP]Patrick loadmorepopup

### PR DESCRIPTION
**Pull request by khoainx**
@minhquan4080

### CODE REVIEW

**Comment by khoainx** [comment]
@minhquan4080 please review my code and give me your opinion!

File: app/assets/javascripts/idea/entrepreneur.js.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+$(document).ready(function() {
+  var currPage = 1;
+  $(window).scroll(function() {
+    if($(window).scrollTop() + $(window).height() > $(document).height() - 200){
+      load_more_entrepreneur(++currPage);
+    }
+  });
+});
```
Comment (lhhieu89) [inline app/assets/javascripts/idea/entrepreneur.js.erb]: 2 files above are the same?

File: app/assets/javascripts/ideaburn.js [hunk 2]
```diff
@@ -37,3 +37,21 @@ $(document).ready(function () {
...
 });
+function load_more_entrepreneur(pageno){
+  $.ajax({
+    url: "/home/loadmoreentrepreneur",
+    data: {pageno: pageno},
+    success: function(result){
+      $(".entrepreneur-content").append(result);
+    }
+  })
+}
```
Comment (lhhieu89) [inline app/assets/javascripts/ideaburn.js]: @@ this one the same too???

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
-    @entrepreneurs = User.entrepreneur.page(1).order('created_at desc')
+    @users = User.order('created_at desc').page(params[:page]).per(8)
+  end
+  def loadmoreentrepreneur
+    pageno = params[:pageno].to_i
+    @users = User.entrepreneur.order('created_at desc').page(params[:page]).per(8)
+    if pageno.nil?
+      @users = @users.page(1).per(8)
+    else
+      @users = @users.page(pageno).per(8)
```
Comment (lhhieu89) [inline app/controllers/home_controller.rb]: I commented the same this one yesterday, right?

**Comment by khoainx** [comment]
@lhhieu89 please review and merge my code to master!

File: app/assets/javascripts/notification.js.erb [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+  var currPage = 1;
+  $("a.next").click(function() {
+    load_more_notification(++currPage);
+  });
+  $(window).scroll(function() {
+    if($(window).scrollTop() + $(window).height() > $(document).height() - 200){
+      load_more_notification(++currPage);
+    }
+  });
+});
```
Comment (minhquan4080) [inline app/assets/javascripts/notification.js.erb]: Please move code to other file.

**Pull request merged**